templates/engage/unlisted/at&t-onboarding.html

```diff
@@ -23,232 +23,193 @@
+        <div class="p-heading-icon__header is-stacked">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/a0f23d1b-Savings.svg" alt="">
+          <h4 class="p-heading-icon__title p-heading--5">AT&amp;T Special Pricing</h4>
+        </div>
+        <p>AT&amp;T can add Ubuntu Pro using the custom SKU in Azure. The AT&amp;T price for Ubuntu Pro is $0.004 per core per hour.</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header is-stacked">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/50dea393-fixes+in+24hrs.svg" alt="">
+          <h4 class="p-heading-icon__title p-heading--5">24/7 SLAed Support</h4>
+        </div>
+        <p>Enterprise-grade support is included with AT&amp;T's Private Offer for Ubuntu Pro instances on Azure.</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header is-stacked">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/d3aa493c-Build-your-first-snap.svg" alt="">
+          <h4 class="p-heading-icon__title p-heading--5">Launched through existing Azure account</h4>
+        </div>
+        <p>The Ubuntu Pro Private Offer can be accessed through the Azure portal when selecting your desired image.</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header is-stacked">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/7953a068-security-1.svg" alt="">
+          <h4 class="p-heading-icon__title p-heading--5">Broadest open source security coverage</h4>
+        </div>
+        <p>Premium custom image that includes a 10 year support commitment from Canonical plus certified components for extended security, compliance, hardening, and Live Kernel patching.</p>
+      </div>
     </div>
   </div>
+</section>
+
+<section class="p-strip">
   <div class="u-fixed-width">
-    <h3 class="u-sv3">Feature Matrix</h3>
-    <table class="p-table">
+    <div class="p-strip u-no-padding--top">
+      <h2>AT&amp;T's Advanced 24/7 SLAed Support</h2>
+      <p>The table below shows the methods for reporting cases, the level of response from Canonical to the case report and the effort period that will be used to resolve the case.</p>
+    </div>
+    <table aria-label="AT&amp;T's advanced 24/7 SLAed support table">
       <thead>
         <tr>
-          <th>Support Feature</th>
-          <th class="u-align--center">AT&T Private Offer Ubuntu Pro w/ Advanced Support</th>
-          <th class="u-align--center">AT&T Private Offer Ubuntu Advantage w/ Advanced Support</th>
+          <th>Severity Level</th>
+          <th>Effort Period</th>
+          <th>Response Method</th>
+          <th>Initial Response Time</th>
+          <th>Subsequent Response Time</th>        
```

Comment:
This tittle is clipped on small screens
<img width="120" alt="image" src="https://user-images.githubusercontent.com/57550290/192310457-8a836e75-278c-47e5-85a6-6d15867348c1.png">

Reply:
Good shout - I've updated with the mobile card pattern to make it more readable.